install-sysdeps.sh

```diff
@@ -7,4 +7,5 @@ sudo apt-get install -y nodejs imagemagick python-sphinx python-yaml \
   python-tz python-virtualenv python-shelltoolbox python-tornado \
   python-gflags g++ xvfb git firefox
 
-sudo npm install -g jshint@2.1.3 mocha-phantomjs@3.3.1 phantomjs@1.9.2-6
+sudo npm install -g jshint@2.1.3 mocha-phantomjs@3.3.1 phantomjs@1.9.2-6 \
+  react-tools
```

Comment:
The charm contains a built rollup archive of the GUI so it doesn't actually have to build anything. This new change may break the `specify a branch` functionality in the charm however. I'll make a card to make the necessary modifications to the charm. 

It's not pinned at a version because they update the tools very rapidly and we need a few bug fixes :)